pkg/test/step.go

```diff
@@ -515,23 +515,24 @@ func (s *Step) LoadYAML(file string) error {
 	return nil
 }
 
-func runtimeObjectsFromPath(s *Step, path string) ([]runtime.Object, error) {
+// RuntimeObjectsFromPath returns an array of runtime.Objects for files / urls provided
+func RuntimeObjectsFromPath(path, dir string) ([]runtime.Object, error) {
 	if http.IsURL(path) {
 		apply, err := http.ToRuntimeObjects(path)
 		if err != nil {
-			return nil, fmt.Errorf("step %q apply %w", s.Name, err)
+			return nil, err
 		}
 		return apply, nil
 	}
 
 	// it's a directory or file
-	paths, err := kfile.FromPath(filepath.Join(s.Dir, path), "*.yaml")
+	paths, err := kfile.FromPath(filepath.Join(dir, path), "*.yaml")
 	if err != nil {
-		return nil, fmt.Errorf("step %q apply %w", s.Name, err)
+		return nil, err
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to find YAML files in %s: %w", filepath.Join(dir, path), err)
```
Often a bare `err` propagated several stack frames up without a comment becomes quite cryptic..